stdlib/public/Darwin/Accelerate/vDSP_FillClearGenerate.swift

```diff
@@ -0,0 +1,416 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+import Accelerate
+
+extension vDSP {
+    
+    /// Fill vector with specified scalar value, single-precision.
+    ///
+    /// - Parameter vector: The vector to fill.
+    /// - Parameter value: The fill value.
+    @inline(__always)
+    @available(iOS 9999, OSX 9999, tvOS 9999, watchOS 9999, *)
+    public static func fill<V>(_ vector: inout V,
```

Comment:
@airspeedswift @moiseev `fill` feels like an API that we should be providing on MutableCollection?